ical.go

```diff
@@ -155,6 +155,11 @@ func (prop *Prop) DateTime(loc *time.Location) (time.Time, error) {
 	return time.Time{}, fmt.Errorf("ical: cannot process: (%q) %s", valueType, prop.Value)
 }
 
+func (prop *Prop) SetDate(t time.Time) {
+	prop.SetValueType(ValueDate)
+	prop.Value = t.Format("20060102")
```

Comment:
Can we use `dateFormat` here instead?

Reply:
Sure. `SetDateTime` can similarly use `datetimeFormat` and `datetimeUTCFormat`. It can be done in a separate commit.

Reply:
Ah, I forgot that these weren't globals. 

> SetDateTime can similarly use datetimeFormat and datetimeUTCFormat. It can be done in a separate commit.

That would be super nice :)